templates/monorepo/README.md

```diff
@@ -0,0 +1,85 @@
+# TSDX Monorepo User Guide
+
+## Usage
+
+This monorepo is setup for a dummy `@mono/` NPM organization. There are 2 packages by default:
+
+- `@mono/react` - A placholder React component
+- `@mono/utils` - A utils packages
+
+Unlike other TSDX templates, the developer experience for this template is currently a bit more manual.
+
+Your first order of business will be to search and replace `@mono` for the npm organization of your own.
+
+After that you can install all the dependencies in the root directory. Since the monorepo uses Lerna and Yarn Workspaces, npm CLI is not supported (only yarn).
+
+```sh
+yarn install
+```
+
+This will install all dependencies in each project, build them, and symlink them via Lerna
+
+## Development workflow
+
+In one terminal, run tsdx watch in parallel:
+
+```sh
+yarn start
+```
+
+This builds each package to `<packages>/<package>/dist` and runs the project in watch mode so any edits you save inside `<packages>/<package>/src` causes a rebuild to `<packages>/<package>/dist`. The results wil stream to to the terminal.
```

Comment:
Just noticed a couple of typos in this paragraph: "The results will stream to the terminal."

Reply:
```suggestion
This builds each package to `<packages>/<package>/dist` and runs the project in watch mode so any edits you save inside `<packages>/<package>/src` cause a rebuild to `<packages>/<package>/dist`. The results will stream to to the terminal.
```